src/ts/binaryen_wasm_ts.js

```diff
@@ -0,0 +1,4 @@
+/* the purpose of this file is simply to enable local compilation of the binaryen_ts.ts file.
```

Comment:
What does "enable local compilation" mean here? Is this tricking some tool into thinking that the generated file always exists?

Reply:
"enable local compilation" means running tsc for binaryen_ts.ts outside the build. This is particularly useful for checking the typescript code during development.